src/libraries/System.Private.CoreLib/src/System/Runtime/Versioning/RequiresPreviewFeaturesAttribute.cs

```diff
@@ -15,7 +15,12 @@ namespace System.Runtime.Versioning
                 AttributeTargets.Property |
                 AttributeTargets.Field |
                 AttributeTargets.Event, Inherited = false)]
-    public sealed class RequiresPreviewFeaturesAttribute : Attribute
+#if SYSTEM_PRIVATE_CORELIB
+    public
+#else
+    internal
+#endif
+        sealed class RequiresPreviewFeaturesAttribute : Attribute
```

Comment:
FYI: @buyaa-n @jeffhandley I need to modify `RequiresPreviewFeaturesAttribute` to not expose new API in netstandard (same way how [`UnsupportedOSPlatformAttribute`](https://github.com/dotnet/runtime/blob/d43c048b3698de191cbeccc3cda65787b4297aac/src/libraries/System.Private.CoreLib/src/System/Runtime/Versioning/PlatformAttributes.cs#L95-L99) in defined.